inlang/source-code/sdk/src/resolve-modules/plugins/types.ts

```diff
@@ -18,7 +18,7 @@ import type { ProjectSettings } from "@inlang/project-settings"
  */
 export type NodeishFilesystemSubset = Pick<
 	NodeishFilesystem,
-	"readFile" | "readdir" | "mkdir" | "writeFile" | "watch"
+	"readFile" | "readdir" | "mkdir" | "writeFile" | "watch" | "rm" | "stat"
 >
```

Comment:
which plugins need access to `rm` and `stat`?

Reply:
Good catch. We need those for file locking - and this was a Problem with the way loadProject was structured when we had both persistency and plugins in this PR should be changed again now.